debug/symbol_table.go

```diff
@@ -0,0 +1,110 @@
+package debug
+
+import (
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type SymbolTable struct {
+	Locations  []Location
+	Functions  []Function
+	mFunctions map[string]int `cbor:"-"` // frame.File+frame.Function to id in Functions
```

Comment:
frame.Function is fully qualified name of the function. Do we need to store also the file name?

Reply:
this is derived from the `profile/` package which is itself derived from go std, don't think it matters too much indeed :)